### PR TITLE
Add docs about running Ktlint

### PR DESCRIPTION
**Pull request by simonschiller**
## Proposed Changes

  - Add docs about running Ktlint

## Testing

Test: N/A, docs change only

## Issues Fixed

Fixes: -

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ cd room
 # Run additional checks
 ./gradlew buildOnServer
 
+# Run coding standard & guideline checks
```
Comment (yigit) [inline CONTRIBUTING.md]: @tikurahul @liutikas 
Do we know why `bOS` doesn't cover this?

Comment (tikurahul) [reply CONTRIBUTING.md]: `ktlint` checks are performed using git hooks in AOSP. These hooks are installed automatically. We should try and do something similar.

Comment (dlam) [reply CONTRIBUTING.md]: We can provide links to the commit hooks we use, but it's impossible to force someone to install them, so we should just add the check to our build imo

Comment (dlam) [reply CONTRIBUTING.md]: Also is ktlint sufficient? Don't we also have error-prone, etc?

Comment (dlam) [inline CONTRIBUTING.md]: Can just delete this header and combine it with the additional checks section.

**Comment by dlam** [comment]
Let me spend a bit of time investigating why ktlint isn't running before merging this - https://issuetracker.google.com/166128980

**Comment by oramirezd** [comment]
Hello!

**Comment by oramirezd** [comment]
I want to download all this file but I can not download all the archive.Just download for example one part of the file.80mb or 90mb and then stop the app to download to I use.

**Comment by oramirezd** [comment]
My phone number it's +5356404000 I am in Watsapp.

**Comment by oramirezd** [comment]
My email it's oramirezd@nauta.cu

**Comment by dlam** [comment]
This PR is superseded by actually integrating ktlint in presubmit here: https://github.com/androidx/androidx/pull/106

**Pull request closed without merging**